shared/src/business/utilities/DateHandler.js

```diff
@@ -432,6 +432,17 @@ const validateDateAndCreateISO = date => {
   }
 };
 
+/**
+ * Subtracts a previous year from passed in ISO date
```

Comment:
Smallest tiniest nit: Should this say "subtracts specified amount of time" instead of "previous year" as it looks like this is configurable?

Reply:
Changed the wording of the comment to better reflect that the function is indeed configurable!